x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/page.tsx

```diff
@@ -67,29 +60,12 @@ export const Page: FC = () => {
                   />
                 </h1>
               </EuiTitle>
-            </EuiPageContentHeaderSection>
-            <EuiPageContentHeaderSection>
-              <EuiFlexGroup alignItems="center">
-                {/* grow={false} fixes IE11 issue with nested flex */}
-                <EuiFlexItem grow={false}>
-                  <RefreshAnalyticsListButton />
-                </EuiFlexItem>
-                {/* grow={false} fixes IE11 issue with nested flex */}
-                <EuiFlexItem grow={false}>
-                  <CreateAnalyticsButton {...createAnalyticsForm} />
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiPageContentHeaderSection>
-          </EuiPageContentHeader>
-          <EuiPageContentBody>
-            <EuiSpacer size="l" />
-            <EuiPanel>
-              <DataFrameAnalyticsList
-                blockRefresh={blockRefresh}
-                openCreateJobModal={createAnalyticsForm.actions.openModal}
-              />
-            </EuiPanel>
-          </EuiPageContentBody>
+            </EuiPageHeaderSection>
+          </EuiPageHeader>
+          <DataFrameAnalyticsList
+            blockRefresh={blockRefresh}
+            createAnalyticsForm={createAnalyticsForm}
```

Comment:
Would it make sense to move `useCreateAnalyticsForm` down into `DataFrameAnalyticsList` so we don't need to pass `createAnalyticsForm` down?

Reply:
it's the first thing I tried, but `useCreateAnalyticsForm` relied on `useKibanaContext`, which is not defined on the management page.

Reply:
Ah, makes sense, let's leave it as it is then.